elastic-search-support/src/test/resources/json/type_mapping_classification_collection_complex_type.json

```diff
@@ -0,0 +1,20 @@
+{
+  "properties":{
+    "classification":securityClassificationMapping,
+    "value":{
+      "properties":{
+        "id":disabledMapping,
+        "classification":{
```

Comment:
Not a big deal but shall we have a space after colon to make it consistent with other json files